### ISSUE TITLE
Choose versioning system

### ISSUE THREAD

**Issue by ChantOfAnAnt** [ChantOfAnAnt/glider#9]
SemVer is good for libraries, frameworks, OS and any other kind of software that end user relies on heavily. That being said, SemVer do no good in continuous development and immediate release workflow for web applications, such as Glider. Numerous other versioning systems exist in wild. So, new versioning system must be selected and introduced to project.

**Comment by ChantOfAnAnt** [comment]
**TL;DR:** Release is just a tradition for software versioning. 

Okay, because of simple gh-pages deployment model and no public API to refer to, there is no use for SemVer.

End user, typically, need to know that application is developed *right now* (at least maintained) and last release was not *too long ago*.

I decided to go with [CalVer](http://calver.org/). Glider versions will be in **YYYY.0M.0D** format. It is release date with zero-padded month and day components.

Development iteration resolves in release and deploy, scheduled in the end of every week. Next release combine all delivered additions, changes and fixes after previous release.

Nothing fancy here, just use natural versioning already made by universe.

**Comment by ChantOfAnAnt** [comment]
Well, this was a bad call. The best approach is to use SemVer for milestones versions and internal bisection. This versions (not technically releases!) will accumulate changes from all commits (builds). And application must be build as quickly as possible. This is great use-case for Travis-CI and Yarn (yeah, I know, more tools), as this will provide reliable builds on developer machines and CI servers.

So, quick list of items to do:

- [ ] Introduce Yarn
- [ ] Reintroduce Travis-CI + write script for incorporating build number to application
